apps/integration/tsconfig.json

```diff
@@ -12,5 +12,16 @@
     {
       "path": "./tsconfig.editor.json"
     }
-  ]
+  ],
+  "compilerOptions": {
+    "forceConsistentCasingInFileNames": true,
+    "strict": false,
+    "noImplicitReturns": false,
+    "noFallthroughCasesInSwitch": true
+  },
+  "angularCompilerOptions": {
+    "strictInjectionParameters": false,
+    "strictInputAccessModifiers": false,
+    "strictTemplates": false
+  }
```

Comment:
Strict mode is disabled by default, so you can likely just remove these altogether? This isn't related to the tabset fix?

Reply:
Nope, it isn't. The `integration` SPA did need updates to the `tsconfig.json`, but these here are just remnants of me playing around to fix the issues I was running into. I'll remove them.